### ISSUE TITLE
 N4 Bias Field Correction

### ISSUE THREAD

**Issue by vasl12** [SimpleITK/SimpleITK#1354]
Hi, on the  N4 Bias Field Correction Python example there is the following code snippet. 

`output = corrector.Execute(image, maskImage)`
`log_bias_field = corrector.GetLogBiasFieldAsImage(inputImage)`
`output = inputImage / sitk.Exp( log_bias_field )`

From these lines we can tell that the output is only influenced by the last line. Am I missing anything? Shouldn't it be that it is also influenced by `output = corrector.Execute(image, maskImage)`?

**Comment by blowekamp** [comment]
The example should be clarified with:
```
corrected_image =  corrector.Execute(image, maskImage)
log_bias_field = corrector.GetLogBiasFieldAsImage(inputImage)
bias_field = inputImage / sitk.Exp( log_bias_field )
```

**Comment by vasl12** [comment]
Hi, thank you for the answer. 
In the code you wrote the corrected_image is still not influencing the result.

**Comment by zivy** [comment]
Hello @vasl12,

The line which saves the corrected image which doesn't appear above needs to be changed to:
```
sitk.WriteImage(corrected_image, sys.argv[2])
```

**Comment by zivy** [comment]
closed by #1379
